app/letters/utils.py

```diff
@@ -175,6 +175,30 @@ def move_sanitised_letter_to_test_or_live_pdf_bucket(filename, is_test_letter, c
     )
 
 
+def move_invalid_templated_letter_to_invalid_bucket(page_count, key_type, reference, created_at, postage):
```

Comment:
This function isn't tested - it's mocked when we're testing the task that calls it. Can you add some tests for the different paths through this code?

Reply:
Thanks - I meant to write a test for that, just got missed.